hoomd_benchmarks/microbenchmark_custom_force.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2021-2022 The Regents of the University of Michigan
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Custom Force benchmark."""
+
+import hoomd
+from . import common
+from .configuration.hard_sphere import make_hard_sphere_configuration
+
+
+class EmptyForce(hoomd.md.force.Custom):
+    """Custom force which calls an empty set_forces method."""
+
+    def __init__(self):
+        super().__init__()
+
+    def set_forces(self, timestep):
+        """Set the forces."""
```

Comment:
The overhead of an empty force should be very low. I would be surprised if it were any different than the trigger overhead.

I would be more interested in the overhead of a constant force set in Python.

Reply:
I'll do a constant force compared to an empty custom force.